.github/workflows/e2e.yaml

```diff
@@ -70,15 +70,14 @@ jobs:
           retention-days: 15
 
       - name: Send Slack alert notification
-        id: slack
         if: ${{ failure() }}
         uses: slackapi/slack-github-action@v2.0.0
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-          SUMMARY_URL: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
         with:
-          channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
-          slack-message: |
-            :x: On repository ${{ github.repository }} the Workflow *${{ github.workflow }}* has failed.
-
-            Details: ${{ env.SUMMARY_URL }}
+          method: chat.postMessage
```

Comment:
Out of scope for this PR: We can consider introducing a more complex flow at some point that starts a thread / updates an existing message for `main` and `release-*` branches.

Reply:
Look like we can "update" a message https://github.com/slackapi/slack-github-action/tree/main?tab=readme-ov-file#updating-a-message

Reply:
ah here it is https://github.com/slackapi/slack-github-action/tree/main?tab=readme-ov-file#replying-to-a-message